### ISSUE TITLE
Shortcut assignment for Enable/Disable Advanced Digitizing not possible

### ISSUE THREAD

**Issue by beyhh** [qgis/QGIS#58679]
### What is the bug or the crash?

It is not possible to assign a shortcut to the Enable/Disable Advanced Digitizing button on the Advanced Digitizing Panel. Can this Enable/Disable toggle button be added to the shortcut list please?

![image](https://github.com/user-attachments/assets/193927fd-bdcb-4228-91c1-dbc6693dd6e6)

### Steps to reproduce the issue

Settings > Keyboard Shortcuts 

Type "Digitizing" into the search field. You will find only an option to assign a shortcut for showing the Advanced Digitizing Panel.

![image](https://github.com/user-attachments/assets/b858280b-71f0-43ca-be80-56185a677e28)

### Versions

QGIS version
3.34.7-Prizren
QGIS code revision
6f7d735c
Qt version
5.15.13
Python version
3.12.3
GDAL/OGR version
3.9.0
PROJ version
9.4.0
EPSG Registry database version
v11.004 (2024-02-24)
GEOS version
3.12.1-CAPI-1.18.1
SQLite version
3.45.1
PDAL version
2.6.3
PostgreSQL client version
16.2
SpatiaLite version
5.1.0
QWT version
6.2.0
QScintilla2 version
2.14.1
OS version
Windows 11 Version 2009

Active Python plugins
db_manager
0.1.20
grassprovider
2.12.99
MetaSearch
0.3.6
processing
2.12.99
shapetools
3.5.1
latlontools
3.7.1

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [X] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by elpaso** [comment]
Yeah, this is annoying. But I honestly see this more a feature request than bug, unfortunately this enhancement requires a lot of internal changes. The same problem exists for all other shortcuts in the advanced digitizing panel.
